code/modules/clothing/head/hat.dm

```diff
@@ -53,9 +53,8 @@
 
 /obj/item/clothing/head/bio_hood/plague
 	name = "plague doctor's hat"
-	desc = "These were once used by plague doctors. Will protect you from exposure to the Pestilence."
+	desc = "These were once used by plague doctors. This hat will only slightly protect you from exposure to the Pestilence."
 	icon_state = "plaguedoctor"
-	clothing_flags = THICKMATERIAL | BLOCK_GAS_SMOKE_EFFECT | SNUG_FIT | STACKABLE_HELMET_EXEMPT
```

Comment:
Because this is a subtype of `/obj/item/clothing/head/bio_hood` this will leave it with
`THICKMATERIAL | BLOCK_GAS_SMOKE_EFFECT | SNUG_FIT | STACKABLE_HELMET_EXEMPT | HEADINTERNALS`

Given that this hat is, I think, just a hat? You probably want `clothing_flags = NONE` instead.

Reply:
Oh.

Reply:
I suspect the coat also possesses these same flags, so any changes to this should be reflected on the coat. But maybe that's approaching a balance change? I don't know.